src/openms/source/ANALYSIS/ID/AScore.cpp

```diff
@@ -135,155 +153,157 @@ namespace OpenMS
         for (Size i = 0; i < 10; ++i)
```

Comment:
Its the one used in the publication. I think there are many ways the implementation can be improved (e.g. make it independent of Phospho). Feature?